docs_user/modules/openstack-tls_everywhere.adoc

```diff
@@ -0,0 +1,203 @@
+[id="tls-everywhere_{context}"]
+
+//:context: tls
+
+//kgilliga: This module will be converted to an assembly. Check xref contexts.
+
+= TLS Everywhere
+
+_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
+
+== Prerequisites
+
+* Check that the source deployment is using TLS Everywhere, skip this step if it's not.
+* Make sure the previous Adoption steps have been performed successfully.
+* Make sure the backend services on the new deployment are not started yet.
+
+== Variables
+
+Define the shell variables used in the following steps. The values are illustrative and refer to a single node standalone director deployment. Use values that are correct for your environment:
+
+----
+IPA_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 podman exec -ti freeipa-server-container"
+----
+
+In this example the FreeIPA instance is running on a separate host, in a container.
+
+== Transfer the CA certificate and key from FreeIPA to the cert-manager Operator
+
+=== Exporting from FreeIPA
+
```

Comment:
This is only going to work if : 

1. The CA that is used for tls-e in osp 16/17 is not used for anything else.  In that case, it may be advisable not to extract the CA cert.  It is the expectation that this CA will be decommissioned afterwards and the new cert-manager CA will be used instead.
2. The IPA CA has its certs stored in nssdb.  Otherwise, if the cert is stored in an HSM, for instance, the key may not be retrievable.  Or if it is retrievable, we may require special HSM utilities.